.travis.yml

```diff
@@ -33,6 +33,13 @@ env:
 matrix:
   fast_finish: true
   include:
+    - env:
+      - HADOLINT="$HOME/hadolint"
+      install:
+      - sudo apt-get install jq
+      - curl -sLo "$HADOLINT" $(curl -s https://api.github.com/repos/hadolint/hadolint/releases/latest | jq -r '.assets | .[] | select(.name=="hadolint-Linux-x86_64") | .browser_download_url') && chmod 700 ${HADOLINT}
+      script:
+      - git ls-files --exclude='*Dockerfile*' --ignored | xargs --max-lines=1 "$HADOLINT"
```

Comment:
> I changed it to only run in one job cause everything else is kinda pointless. The Dockerfiles don't change with the php version.

I wasn't suggesting to run it on every job. On second look I'd move this and the shellchecks into the jshint node_js container down below to make it the dedicated non-PHP lint job.

Reply:
Good idea and already done.